cmd/scepserver/scepserver.go

```diff
@@ -167,7 +168,10 @@ func main() {
 		e := scepserver.MakeServerEndpoints(svc)
 		e.GetEndpoint = scepserver.EndpointLoggingMiddleware(lginfo)(e.GetEndpoint)
 		e.PostEndpoint = scepserver.EndpointLoggingMiddleware(lginfo)(e.PostEndpoint)
-		h = scepserver.MakeHTTPHandler(e, svc, log.With(lginfo, "component", "http"))
+		scepHandler := scepserver.MakeHTTPHandler(e, svc, log.With(lginfo, "component", "http"))
+		r := mux.NewRouter()
+		r.Handle("/scep", scepHandler)
+		h = r
```

Comment:
This `cmd/scepserver/scepserver.go` file was just for manual testing, right?

Reply:
hmm nope, I didn't actually change this file...